scylla/src/frame/cql_types_test.rs

```diff
@@ -0,0 +1,47 @@
+// TODO: Requires a running local Scylla instance
+#[tokio::test]
+#[ignore]
+async fn test_cql_collections() {
+    // Create connection
+    let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+
+    println!("Connecting to {} ...", uri);
+
+    let session = crate::transport::session::Session::connect(uri, None)
+        .await
+        .unwrap();
```

Comment:
We are now using `SessionBuilder` to construct sessions. The example won't compile after merging this branch to `main` branch. Please rebase onto current `main` and fix.

Reply:
Fixed.